### ISSUE TITLE
`Date.now()` expression returns incorrect result

### ISSUE THREAD

**Issue by Taritsyn** [Microsoft/ClearScript#34]
Hello!

After upgrading to version 5.5.1, `V8ScriptEngine.Evaluate` method began to return an incorrect result of the `Date.now()` expression. Current result (`16336909`) differs significantly from the result of version 5.5.0 (`1514915269409`).

For verification, you can use the following code:

```csharp
using System;
using Microsoft.ClearScript.V8;

namespace TestClearScript
{
	class Program
	{
		static void Main(string[] args)
		{
			using (var engine = new V8ScriptEngine())
			{
				double totalMilliseconds = Convert.ToDouble(engine.Evaluate("Date.now();"));
				Console.WriteLine("totalMilliseconds = {0}", totalMilliseconds);
			}
		}
	}
}
```

**Comment by ClearScriptLib** [comment]
Confirmed. Thanks, Andrey!

Unlike the previous stable version, V8 6.3 calls the host to read the wall clock, and ClearScript's implementation of that callback is broken. We'll post a fix shortly.

Thanks again!

**Comment by ClearScriptLib** [comment]
Fixed in [Version 5.5.1.1](https://github.com/Microsoft/ClearScript/releases/tag/5.5.1.1).

**Comment by Taritsyn** [comment]
Thanks!
